src/commands/testrail.ts

```diff
@@ -111,10 +111,10 @@ class JahiaTestrailReporter extends Command {
   async run() {
     const {flags} = this.parse(JahiaTestrailReporter)
 
-    if (flags.runName === 'Automated Execution - ') {
+    if (flags.runName === 'AE - ') {
       const date = new Date()
       const format = 'YYYY-MM-DD HH:mm:ss [GMT]Z (z)'
-      const output = formatToTimeZone(date, format, {
+      const output = flags.projectName+'-'+formatToTimeZone(date, format, {
```

Comment:
this part was not super clean in the first place, since output is only used once, I'd suggest doing it like this instead:

```javascript
const runDate = formatToTimeZone(date, format, {
    timeZone: 'Europe/Paris',
 })
flags.runName += `${flags.projectName}-${runDate}`
```